discrete_env.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+
+from gym import Env, spaces
+from gym.utils import seeding
+
+
+def categorical_sample(prob_n, np_random):
+    """
+    Sample from categorical distribution
+    Each row specifies class probabilities
+    """
+    prob_n = np.asarray(prob_n)
+    csprob_n = np.cumsum(prob_n)
+    return (csprob_n > np_random.rand()).argmax()
+
+
+class DiscreteEnv(Env):
+
+    """
+    Has the following members
```

Comment:
Please follow the [docstring format in FastForest for classes](https://github.com/ThrunGroup/FastForest/blob/main/data_structures/wrappers/extremely_random_forest_classifier.py#L8-L16)